### PR TITLE
Add registration success redirect

### LINKED ISSUE #79: redirect user to `event/register/success` on successful registration
# Expected Behavior

when user successfully registers for the event 
 - redirect user to  `/register/success` (this page uses the [template for 404 page](https://github.com/srm-kzilla/srmkzilla-net/blob/main/pages/404.tsx)) 
 - after 3 seconds redirect the user from `/register/success` to  `/events`

# Current Behavior

- upon successful registration the website redirects to `srmkzilla.net` which is ambiguous for the user to understand whether his/her registration is successful or not.

### PR DESCRIPTION
**Pull request by alvinbengeorge**
# Description

Added a new page just to display a successful message to avoid confusion after the user registered for an event
image
Fixes #79 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Used web to check if the automatic redirect happens after 3 seconds

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: pages/register/index.tsx [hunk 1]
```diff
@@ -4,11 +4,12 @@ import Router from 'next/router'
...
 import 'react-toastify/dist/ReactToastify.css'
-import Footer from '../shared/components/footer'
-import { getEvent, register } from '../utils/api'
-import { registerFormData, registerValidationSchema } from '../utils/schema'
+import Footer from '../../shared/components/footer'
+import { getEvent, register } from '../../utils/api'
+import { registerFormData, registerValidationSchema } from '../../utils/schema'
 import { useReward } from 'react-rewards'
 import { IoInformationCircleOutline } from 'react-icons/io5'
+import { encode } from 'querystring'
```
Comment (shawshankkumar) [inline pages/register/index.tsx:12]: why do we need this?

File: pages/register/success.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { motion } from 'framer-motion'
+import Head from 'next/head'
+import Image from 'next/image'
+
+const Success = () => {
+  const router = useRouter()
+  const { title } = router.query
+  useEffect(() => {
+    setTimeout(() => {
+      router.push('/events')
```
Comment (shawshankkumar) [inline pages/register/success.tsx:12]: dont' push. leave it on this page.

**Review by shawshankkumar — CHANGES_REQUESTED** [review]

**Comment by shawshankkumar** [comment]
@alvinbengeorge @Manaya20 ?

File: pages/register/success.tsx (hunk 2 above)
Comment (alvinbengeorge) [reply pages/register/success.tsx:12]: Utkarsh bhaiya only said that, wait for 3 seconds and go to events

File: pages/register/index.tsx [hunk 3]
```diff
@@ -79,7 +79,7 @@ const Register = ({
         )
         reward()
         setTimeout(() => {
-          Router.push('/events')
+          Router.push('/register/success/?title=' + encodeURIComponent(title)) // redirects to the success page
```
Comment (shawshankkumar) [inline pages/register/index.tsx:82]: remove the comment.

File: pages/register/success.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react'
+import { useRouter } from 'next/router'
+import { motion } from 'framer-motion'
+import Head from 'next/head'
+import Image from 'next/image'
+
+const Success = () => {
+  const router = useRouter()
+  const { title } = router.query
+
```
Comment (shawshankkumar) [inline pages/register/success.tsx:10]: unnecessary empty lines

**Review by shawshankkumar — CHANGES_REQUESTED** [review]
Why do we have the success.png file?

**Review by shawshankkumar — APPROVED** [review]
LGTM! Great work @Manaya20 @alvinbengeorge

**Pull request merged**